docs/topics/crud-mission-resources.adoc

```diff
@@ -1,5 +1,5 @@
 [[about-crud]]
-= CRUD resources
+= Relational Database Resources
 
 More background and related information on CURD, HTTP API and REST can be found here:
```

Comment:
CURD -> CRUD

Or maybe add something like "More background and related information on running relational databases in OpenShift, CRUD, HTTP API and REST can be found here"? That is, assuming we do provide some links to documentation on deploying relational databases. Which we should, I filed an issue on that somewhere.